bittensor_cli/cli.py

```diff
@@ -2817,21 +2827,24 @@ def stake_get_children(
         The command compiles a table showing:
 
         - ChildHotkey: The hotkey associated with the child.
-
         - ParentHotKey: The hotkey associated with the parent.
-
         - Proportion: The proportion that is assigned to them.
-
         - Expiration: The expiration of the hotkey.
 
-
         # Example usage:
 
         [green]$[/green]btcli stake get_children --netuid 1
+        [green]$[/green]btcli stake get_children --all-netuids
 
-        [italic]]Note[/italic]: This command is for users who wish to see child hotkeys among different neurons (hotkeys) on the network.
+        [italic]Note[/italic]: This command is for users who wish to see child hotkeys among different neurons (hotkeys) on the network.
         """
         wallet = self.wallet_ask(wallet_name, wallet_path, wallet_hotkey)
+        if all_netuids:
+            netuid = None
+        elif not netuid:
+            netuid = Prompt.ask("Netuid")
```

Comment:
For all of the netuid(int) questions here, 

If you do not enter netuid in the args, it will prompt you for netuid. 
If you enter 'all' it should behave as if you entered the `--all` flag to get the children from all subnets.